src/main/main.ts

```diff
@@ -23,7 +23,7 @@ app.on('ready', () => {
   createMainMenu(Menu, createMenuTemplate(app));
   initializeIpcEvents();
 
-  if (process.argv[2]) {
+  if (process.argv[2] && process.argv[2] != '--no-dev-tools') {
```

Comment:
You might want to do some fancier command line parsing here. I opted for getting something working.

Reply:
Can we just re-use the code above from `WindowManager`? I'm actually already using `argv[2]` for scene-name, so it'd be optimal if `--no-dev-tools` could be any argument index.